.eslintrc.yml

```
@@ -0,0 +1,3 @@
+extends: cheminfo-typescript
+rules:
+  no-dupe-class-members: off
```

```suggestion
  no-dupe-class-members: off

```

Btw why is this rule turned off?